outputs.tf

```diff
@@ -70,3 +70,8 @@ output "keys" {
   sensitive   = true
   value       = { for k, v in local.names : k => var.generate_keys ? base64decode(google_service_account_key.keys[v].private_key) : "" }
 }
+
+output "display_names" {
+  description = "display names variable."
+  value       = var.display_names
+}
```

Comment:
I don't think redirecting this as an output as value since users can get it from service_accounts_map if needed.

Reply:
@bharathkkb 
Thanks for the review!
Yes, that was exactly what you said.
I was able to verify the display_name in the service_accounts_map, so I removed it from the outputs.tf.

Reply:
@bharathkkb I fixed that. Thanks.

Reply:
@bharathkkb 
I fixed that.